src/Template/Element/status_table.ctp

```diff
@@ -58,6 +59,10 @@ $sensorStatuses->some(function ($sensorStatus) {
                 $lastExecuted,
                 ['class' => $tableClass],
             ],
+            [
+                $wasCheckFromCache ? 'Cached' : 'Not cached',
+                ['class' => $wasCheckFromCache ? 'success' : 'info']
```

Comment:
Not sure these classes make sense in that case....

Reply:
You're probably right. They are mostly decorative, and when the decoration is unclear, there's no point. Should I commit a change without assigning a class to the column.